rero_ils/modules/documents/listener.py

```diff
@@ -156,3 +158,22 @@ def enrich_document_data(sender, json=None, record=None, index=None,
             'doc', document_pid)
         if local_fields:
             json['local_fields'] = local_fields
+        # index both ISBN 10 and 13 format
+
+        def filter_isbn(identified_by):
+            """Filter identified_by for type bf:Isbn."""
+            return identified_by.get('type') == 'bf:Isbn'
+
+        filtered_identified_by = filter(
+            filter_isbn,
+            json.get('identifiedBy', [])
+        )
+        isbns = set()
+        for identified_by in filtered_identified_by:
+            isbn = identified_by['value']
+            isbns.add(isbn)
+            if is_isbn10(isbn):
+                isbns.add(to_isbn13(isbn))
+            elif is_isbn13(isbn):
+                isbns.add(to_isbn10(isbn))
+        json['isbn'] = list(isbns)
```

Comment:
Do you need to to check if the `isbns` is empty here before adding it to `json`?

Reply:
I think not, this is only in ES, if it is empty we can search ist as expected.